source/module_base/memory.cpp

```diff
@@ -308,7 +308,7 @@ void Memory::print_all(std::ofstream &ofs)
 //             << std::setw(20) << class_name[k]
 //             << std::setw(20) << name[k]
 //             << std::setw(15) << consume[k];
-    std::cout<<"\n ----------------------------------------------------------"<<std::endl;
+    //std::cout<<"\n ----------------------------------------------------------"<<std::endl;
```

Comment:
This line has appeared below the time statistics table of std::out for long time, I fix it by the way.